users/views.py

```diff
@@ -69,7 +75,7 @@ def form_valid(self, form):
                 return super().form_invalid(form)
         else:
             form.save()
-        return redirect(reverse('home'))
+        return redirect(reverse('profile', args=[self.request.user.id]))
```

Comment:
is it necessary to use two methods with the same purpose, instead of one?
 how about something like: reverse(view_name, *args, **kwargs)